### PR TITLE
Add name for home breadcrumb in filepicker

### PR DESCRIPTION
**Pull request by Pytal**
- For https://github.com/nextcloud/server/issues/36714

## Summary

Add name for the home breadcrumb shown below

image

This can be read by screen readers

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by artonge — APPROVED** [review]

File: core/css/styles.scss [hunk 1]
```diff
@@ -647,6 +647,7 @@ code {
 		padding-right: 140px;
 
 		div:first-child a {
+			font-size: 0px;
```
Comment (szaimen) [inline core/css/styles.scss:650]: this looks like a hack to me but I suppose there is now better way?

Comment (Pytal) [reply core/css/styles.scss:650]: Yes, the simplest solution for now

**Review by szaimen — APPROVED** [review]
Fine by me then but didnt test

**Review by JuliaKirschenheuter — APPROVED** [review]

**Pull request merged**